test/framework/integration_framework/iroha_instance.cpp

```diff
@@ -25,9 +25,11 @@ using namespace std::chrono_literals;
 
 namespace integration_framework {
 
-  IrohaInstance::IrohaInstance(bool mst_support, const std::string &block_store_path)
+  IrohaInstance::IrohaInstance(bool mst_support,
+                               const std::string &block_store_path,
+                               const boost::optional<std::string> &dbname)
```

Comment:
May be use overloading with and without dbname instead of boost::optional?

Reply:
what's the profit?